compiler/llvm/clangUtil.cpp

```diff
@@ -2469,7 +2469,11 @@ void setupForGlobalToWide(void) {
 #if HAVE_LLVM_VER < 50
                           , NULL
 #endif
+#if HAVE_LLVM_VER < 80
                           );
+#else
+                          ).getCallee();
```

Comment:
@mppf - I just updated to master and found that this code does not compile for LLVM 8.  The `.getCallee()` should be `->getCallee()` and similarly with the other .function() calls.  I get the same failure on Mac Homebrew LLVM 8 and Ubuntu LLVM 8.

Reply:
It needs to be .getCallee for LLVM 9 (pre-release). I havn't been able to reproduce your issue. Can you prepare a PR?

Reply:
(I havn't tried to build with LLVM 8 myself yet but it's probably faster for you to fix than for me to get that going)